src/styles/blocks.scss/day.scss

```
@@ -0,0 +1,50 @@
+@import "../utils/_variables";
+
+.day {
+  width: $width;
+  height: $width;
+  background-color: #eee;
+  border: solid black 1px;
+  cursor: pointer;
+  transition: background-color 0.5s, transform 0.5s;
+
+  &::before {
+    content: "";
+    position: absolute;
+    top: 34px;
+    left: 42px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+  }
+
+  &:hover {
+    background-color: pink;
+    transform: translate(0, -20px);
+  }
+
+  &:nth-child(n + 10)::before {
+    left: 33px;
+  }
+}
+
+@for $index from 1 through 31 {
+  .day_day-#{$index}::before {
+    content: "#{$index}";
+    position: relative;
+  }
+}
+
+$days: ("mon": "1", "tue": "2", "wed":"3", "thu": "4", "fri": "5",
+"sat":"6", "sun":"7");
+
+@each $day, $number in $days {
+  .day_start-day_#{$day} {
+    margin-left: calc(101px * (#{$number} - 1));
```

винеси в змінні 101 піксель
```suggestion
    margin-left: calc(($day_size + $gap) * #{$number});
```